### PR TITLE
Random color

### PR DESCRIPTION
**Pull request by Aleksalje**
Added random generator for colors
hex, rgb, rgba, hsl and hsla options

### CODE REVIEW

**Comment by MilosPaunovic** [comment]
Hello @Aleksalje,

First of all, great job!

Next, there are a few small issues - while you were working on the code for this PR, I've added ESLint to codebase; you should pull latest changes from develop branch and run lint command locally (and fix all potential issues).

There is a merge conflict with `package-lock.json` file - it's because of the linter. I suggest for you to delete both `node_modules` and `package-lock.json`, after which you can run `npm i` and it will create new, proper file without conflicts.

Other stuff I've noticed will be comments on the code itself, just for simplicity.

File: README.md [hunk 1]
```diff
@@ -44,6 +44,25 @@ randomer.STRING.NAMES(howMany);
...
+randomer.COLOR.RGBA();
+// return example: rgb(22 26 189 / 60%)
+// random hsl color
+randomer.COLOR.HSL();
+// return example: hsl(346 78% 49%)
+// random hsla color
+randomer.COLOR.HSLA();
+// return example: hsl(134 75% 74% / 26%)
+```
+
```
Comment (MilosPaunovic) [inline README.md:65]: Minor style changes for this part of README.md file; could you please change it?

#### Color
```js
// Hex color
randomer.COLOR.HEX(); // #41D831

// RGB color
randomer.COLOR.RGB(); // rgb(180 206 38)

// RGBA color
randomer.COLOR.RGBA(); // rgb(22 26 189 / 60%)

// HSL color
randomer.COLOR.HSL(); // hsl(346 78% 49%)

// HSLA color
randomer.COLOR.HSLA(); // hsl(134 75% 74% / 26%)
```

File: modules/color.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+const { INTEGER } = require('./number');
+const rndNum = (min = 0, max = 255) => INTEGER(min, max); //Math.round(Math.random() * 255);
```
Comment (MilosPaunovic) [inline modules/color.js:2]: There is no need to shorten names of constants, in this case; we should keep it readable as possible, so, could you just find and replace all `rndNum` to `randomNumber`?

And one more thing, is there a reason why that line is not like this bellow?
`const randomNumber = INTEGER(0, 255)`

**Review by MilosPaunovic — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Aleksalje) [reply README.md]: This has been updated as requested.

File: modules/color.js (hunk 2 above)
Comment (Aleksalje) [reply modules/color.js]: Random color generation has changed so that this is no longer needed.

**Review by MilosPaunovic — APPROVED** [review]

**Pull request merged**